### DISCUSSION TITLE
Nameservers vs. A-Record DNS

### DISCUSSION THREAD

**Discussion by lemmon** [vercel/vercel#5429] [Help] [answered]
Hi,

I've noticed that there are several ways to setup a custom domain with Vercel. Are there any advantages or disadvantages when using Vercel's Nameservers vs. using A-Record DNS only?

Thanks!

**Comment by FreshP0325** [comment, marked as answer]
Hi there,

To a certain extent,there is no distinct different between using Vercel DNS or A record for me - both of them just play a role in routing your domain name to the certain IP address 76.76.21.21.Now we don't have to set CNAME records so there isn't much performance loss without resolving any additional CNAME records.

In fact,the most obvious advantage of using Vercel DNS is that they are right vanity DNS under Constellix's network.So if you are using Godaddy DNS or Namecheap DNS these default DNS provided by registrars,you can have **less DNS resolving time** since Constellix has deployed anycast network through 6 continents and many countries after moving your DNS to Vercel.That means your websites perform better worldwide.However,they don't support DNSSEC and Geo-routing so if you are using these functions with Route53,NSONE and Dyn,while you don't want to lose them,stay there.
